### ISSUE TITLE
Customize website page titles

### ISSUE THREAD

**Issue by seeM** [quarto-dev/quarto-cli#5253]
IIUC posts rendered as website pages have titles always in the format: `<website title> - <post title>`. I would like the option to customize this.

E.g. for my personal website, social preview cards render like so:

<img width="610" alt="image" src="https://user-images.githubusercontent.com/559360/233345684-da767bd9-c4d0-43d0-97a6-277018312cf8.png">

But I'd prefer if the `Wasim Lorgat - ` part weren't there, since the website URL already has my name 😄

**Comment by cderv** [comment]
You are referring to Twitter Card or Open Graph based social card, right ? 

Did you already customized the values per our doc ? https://quarto.org/docs/reference/projects/websites.html#social

By default the title of the page will be used, but you can overwrite using those configuration.

**Comment by seeM** [comment]
I’m referring to the same title that seems to be used for the HTML web page and the Twitter and open graph cards. I could override each of those for every single post but I’d prefer if it were possible to change the string template once e.g in my `_quarto.yml`.

**Comment by mcanouil** [comment]
Related to #1736 in some aspect.

In `_quarto.yml`, you can add `pagetitle: "{{< meta title >}}"` to overwrite `pagetitle` and remove the website title prefix.

@cderv about `title-prefix` there is currently an issue, see #5196.

**Comment by cderv** [comment]
Good catch for the issue. It seems it is not working as I expected or as it should... 
We'll follow that in #5196. 

@mcanouil workaround is a good one for your need @seeM, I believe it answers your problem, isn't it ?

**Comment by allenmanning** [comment]
@cderv @mcanouil Should we close or move this issue?
